LoRaEngine/LoraKeysManagerFacade/DeviceGetter.cs

```diff
@@ -107,16 +108,16 @@ public async Task<List<IoTHubDeviceInfo>> GetDeviceList(string devEUI, string ga
                     if (await deviceCache.TryToLockAsync())
                     {
                         deviceCache.ClearCache(); // clear the fcnt up/down after the join
-                        log?.LogDebug("Removed key '{key}':{gwid}", devEUI, gatewayId);
+                        log?.LogDebug("Removed key '{key}':{gwid}", someDevEui, gatewayId);
                     }
                     else
                     {
-                        log?.LogWarning("Failed to acquire lock for '{key}'", devEUI);
+                        log?.LogWarning("Failed to acquire lock for '{key}'", someDevEui);
                     }
                 }
                 else
                 {
-                    log?.LogDebug("dev nonce already used. Ignore request '{key}':{gwid}", devEUI, gatewayId);
+                    log?.LogDebug("dev nonce already used. Ignore request '{key}':{gwid}", someDevEui, gatewayId);
```

Comment:
The default formatting for the ToString on the Eui is using '-' as separators. That's a change from the previous formatting, which was not using any separators. We should consider keeping the default to not change that. One thing that is easier if we do not have the separators is copying around the value.

Reply:
This was addressed with 6774e906970a25a27d7e163acf4f545376e454d9